sql/item_mods_pet.sql

```diff
@@ -96,6 +96,7 @@ INSERT INTO `item_mods_pet` VALUES (11720,28,3,3);      -- Automaton: "Magic Atk
 INSERT INTO `item_mods_pet` VALUES (11739,27,2,1);      -- Caller's Sash Avatar: Enmity +2
 INSERT INTO `item_mods_pet` VALUES (11739,28,2,1);      -- Avatar: Magic Atk. Bonus +2
 INSERT INTO `item_mods_pet` VALUES (12520,27,-3,1);     -- Evoker's Horn Avatar: Enmity -3
+INSERT INTO `item_mods_pet` VALUES (12649,370,3,2);     -- Drachen Mail Adds "Regen" effect to wyvern
```

Comment:
1 hp = 1 tic, ignore the 3 second comment on bg, since a tick is 3 seconds...

http://ffxiclopedia.wikia.com/wiki/Tick:
> The standard tick occurs every 3 seconds (Earth time). This is used to trigger the effects of DOT spells such as Poison and Choke as well as recovery effects such as Regen, Refresh, and Regain. Such effects generally last for a certain predetermined amount of ticks before wearing off.

and from BG: 
> The regen effect is 1 HP every 3 seconds.